File: OpenMP/solutions/omp_firstprivate.f
      program threadnum
              
        use omp_lib
        implicit none

        integer :: var = 5
!$OMP PARALLEL FIRSTPRIVATE(var)
        var = var + omp_get_thread_num(); 
        print *,"Hello World from thread = ",omp_get_thread_num(),
     &          ": var is ",var
!$OMP END PARALLEL
        
      end program

File: OpenMP/exercises/omp_firstprivate.c
#include <omp.h>
#include <stdio.h>

int main (int argc, char *argv[])
{
  int var = 5;
#pragma omp parallel private(var)
{
  var = var + omp_get_thread_num();
  printf("Hello World from thread = %d: var is %d\n", omp_get_thread_num(), var);
}
}
